src/transformState.js

```diff
@@ -3,9 +3,35 @@
 /**
  * @param {Object} state
  * @param {Object[]} actions
- */
+*/
+
 function transformState(state, actions) {
-  // write code here
+  const stateModified = state;
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(stateModified, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete stateModified[key];
+        }
+        break;
+
+      case 'clear':
+        for (const key in stateModified) {
+          delete stateModified[key];
+        }
+        break;
+
+      case 'default':
```

Comment:
Replace 'case' with 'default' for the error handling block in the switch statement.

[A `default` clause -  if provided, this clause is executed if the value of expression doesn't match any of the case clauses. A switch statement can only have one default clause.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)

Reply:
Sorry, forgot the syntax of the switch statement🤦